rotkehlchen/data/importer.py

```diff
@@ -61,7 +62,7 @@ def exchange_row_to_location(entry: str) -> Location:
     if entry == 'Coinbase':
         return Location.COINBASE
     # TODO: Check if this is the correct string for CoinbasePro from cointracking
-    if entry == 'CoinbasePro':
+    if entry in ('CoinbasePro', 'GDAX'):
```

Comment:
Hehe ... remove the TODO from above. It probably means that it was not the right string. They use GDAX. But keep coinbasepro there just in case ..